prow/cmd/deck/static/tide-history/tide-history.ts

```diff
@@ -182,12 +188,10 @@ function redraw(): void {
   const hist: {[key: string]: Record[]} = typeof tideHistory !== 'undefined' ? tideHistory.History : {};
   const poolKeys = Object.keys(hist);
   for (const poolKey of poolKeys) {
-    const match = RegExp('(.*?):(.*)').exec(poolKey);
```

Comment:
This regex is not exactly the same as `RegExp('(((https|http)://[^/]*/)?.*?)` but I guess functionally they are still equivalent? Do we have tests that ensure we're not breaking anything here?

Reply:
I have done runlocal and at least it works for github